spec/test_app/app/controllers/basechurch/v1/announcements_controller_spec.rb

```diff
@@ -140,7 +151,7 @@
     it_behaves_like 'a response containing an announcement'
   end
 
-  describe 'PUT /announcements/:id' do
+  describe 'PATCH /announcements/:id' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.